documentation/src/main/asciidoc/topics/upgrading.adoc

```diff
@@ -413,9 +417,9 @@ The following changes apply to the Infinispan 13 configuration schemas:
 * The `single-file-store` element has been added to the `urn:infinispan:config` namespace if you want to configure a single file store, which is deprecated.
 * The `endpoints` element in the `urn:infinispan:server` namespace is no longer a repeating element, but a wrapper for 0 or more `endpoint` elements.
 
-=== Migrating Soft-Index File Stores
+==== Migrating Soft-Index File Stores
 
-==== Declarative
+===== Declarative
```

Comment:
Because it's nested too deep, it renders it as normal text. Is it OK?

Reply:
Now:

![image](https://github.com/infinispan/infinispan/assets/13581903/7d128512-00a4-4dd3-a7c6-1d7d6ef060f0)


Before:

![image](https://github.com/infinispan/infinispan/assets/13581903/6c7eb261-ca5f-4adc-9ed7-232dafe933de)

Reply:
Yeah, I think it's fine